pkg/util/deployment/deployment.go

```diff
@@ -40,6 +40,12 @@ import (
 const (
 	// The revision annotation of a deployment's replica sets which records its rollout sequence
 	RevisionAnnotation = "deployment.kubernetes.io/revision"
+	// DesiredReplicasAnnotation is the desired replicas for a deployment recorded as an annotation
+	// in its replica sets.
+	DesiredReplicasAnnotation = "deployment.kubernetes.io/desired-replicas"
+	// TotalReplicasAnnotation is the total replicas for a deployment (desired+maxSurge in case the
+	// new replica set is not satured) recorded in its replica sets.
+	TotalReplicasAnnotation = "deployment.kubernetes.io/total-replicas"
```

Comment:
This is different fro Desired because it is ActualObserved right?  Maybe ParentTotalActualObservedReplicas or ParentTotalActualReplicas or ParentTotalObservedReplicas